pyblish_lite/control.py

```diff
@@ -37,15 +34,26 @@ class Controller(QtCore.QObject):
     # Emitted when processing has finished
     was_finished = QtCore.Signal()
 
+    PART_COLLECT = 'collect'
```

Comment:
These should not be hardcoded.

The "collection" order is already defined by the non-inclusive range `CollectorOrder -> CollectorOrder + 0.5`. You can can use to determine whether a given plug-in is a collector or now, e.g. `plugin_type(Plugin) == "collector"`

Reply:
It's not hardcoded that is constant which is used as key for dictionary. Don't know how to do it without this kind of separation. `plugin_type(Plugin) == "collector"` this works but can be done after pyblish plugin filtering which is too late....